gm4_chairs/data/chairs/functions/kill.mcfunction

```diff
@@ -0,0 +1,7 @@
+# @s = @e[tag=gm4_chairs] and NOT inside stair blocks
+#execute at @s align xyz run tp @e[nbt={RootVehicle:{Entity:{id:"minecraft:pig"}}},dx=0,dz=0,dy=2] ~ ~.4 ~

```

Comment:
Why would you want to tp the player and why wouldn't you use `@p` or something. Also why is this in a comment

Reply:
That command would get the player off the saddle, so if you break the stairs you get right where you were before (and again, about half a block up to prevent you going into the block underneath).
@e was because I was trying something with entities as well and commented it out if I wanted to do something with it later, should've left it out for this upload